posts/blob/master/posts/inside-rust/2019-12-20-governance-wg-meeting.md

```diff
@@ -0,0 +1,28 @@
+layout: post
```

Comment:
whoops wait a minute, @valgrimm, I see that this post was kind of in the wrong place :) I'll patch it...